libs/deal-ticket/src/components/market-info/info-key-value-table.tsx

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import {
+  t,
+  addDecimalsFormatNumber,
+  formatNumberPercentage,
+  formatNumber,
+} from '@vegaprotocol/react-helpers';
+import {
+  KeyValueTableRow,
+  KeyValueTable,
+  Tooltip,
+} from '@vegaprotocol/ui-toolkit';
+import BigNumber from 'bignumber.js';
+import omit from 'lodash/omit';
+import startCase from 'lodash/startCase';
+import type { ReactNode } from 'react';
+import { tooltipMapping } from './tooltip-mapping';
+
+interface RowProps {
+  field: string;
+  value: any;
+  decimalPlaces?: number;
+  asPercentage?: boolean;
+  unformatted?: boolean;
+  assetSymbol?: string;
+}
+
+const Row = ({
+  field,
+  value,
+  decimalPlaces,
+  asPercentage,
+  unformatted,
+  assetSymbol = '',
+}: RowProps) => {
+  const isNumber = typeof value === 'number' || !isNaN(Number(value));
+  const isPrimitive = typeof value === 'string' || isNumber;
+  const className = 'text-black dark:text-white text-ui !px-0 !font-normal';
+  if (isPrimitive) {
```

Comment:
question (non blocking): Not sure why we need this check. I think it stems from some any types defined in this file.

Reply:
https://github.com/vegaprotocol/frontend-monorepo/pull/1133/files#r961402382